RevenueCat.podspec

```diff
@@ -0,0 +1,28 @@
+Pod::Spec.new do |s|
+  s.name             = "RevenueCat"
+  s.version          = "3.13.0-SNAPSHOT"
```

Comment:
we might as well update this to `4.0.0-SNAPSHOT` at this point

Reply:
I was thinking that our automation was going to increment that so I shouldn't update it. That being said, I don't actually know if that gets automatically incremented during deploy 😬

Reply:
😄 
you can do it at any time, just run `bundle exec fastlane bump version:4.0.0-SNAPSHOT`. 
It can be a separate PR, though

Reply:
OOOoooooo ok. Will do that in another PR.